docs/admin_api/rooms.md

```diff
@@ -11,8 +11,21 @@ The following query parameters are available:
 * `from` - Offset in the returned list. Defaults to `0`.
 * `limit` - Maximum amount of rooms to return. Defaults to `100`.
 * `order_by` - The method in which to sort the returned list of rooms. Valid values are:
-  - `alphabetical` - Rooms are ordered alphabetically by room name. This is the default.
-  - `size` - Rooms are ordered by the number of members. Largest to smallest.
+  - `alphabetical` - Same like `name`. This is deprecated.
+  - `size` - Same like `joined_members`. This is deprecated.
+  - `name` - Rooms are ordered alphabetically by room name. This is the default.
+  - `canonical_alias` - Rooms are ordered alphabetically by main alias address of the room.
+  - `joined_members` - Rooms are ordered by the number of members. Largest to smallest.
+  - `joined_local_members` - Rooms are ordered by the number of local members. Largest to smallest.
+  - `version` - Rooms are ordered by room version. Largest to smallest.
+  - `creator` - Rooms are ordered alphabetically by creator of the room.
+  - `encryption` - Rooms are ordered by encryption of the room.
```

Comment:
This is the encryption algorithm, I think? Is it done alphabetically?

Reply:
Values like `m.megolm.v1.aes-sha2` or `NULL`. Yes, it done alphabetically.

Reply:
Please update this comment with that information, e.g.:

> Rooms are ordered alphabetically by the end-to-end encryption algorithm.